src/index.html

```diff
@@ -1,22 +1,276 @@
+    </section>
+
+    <section class="explore-online" id="explore-online">
+      <h2>More to explore online</h2>
+      <div class="explore-online__photos">
+        <a href="#" class="explore-online__link">
+          <img
+          class="explore-online__photo"
+          src="images/image8-square.png"
+          alt="image">
+        </a>
+        <div class="explore-online__text">
+          <h3>Collection Areas</h3>
+          <p>These collection areas study, exhibit, and care for the objects in The Met collection.</p>
+        </div>
+      </div>
+      <div class="explore-online__photos">
+        <a href="#" class="explore-online__link">
+          <img
+          class="explore-online__photo"
+          src="images/image9-square.png"
+          alt="image">
+        </a>
+        <div class="explore-online__text">
+          <h3>Travel</h3>
+          <p>Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.</p>
+        </div>
+      </div>
+      <div class="explore-online__photos">
+        <a href="#" class="explore-online__link">
+          <img
+          class="explore-online__photo"
+          src="./images/image10-square.png"
+          alt="image">
+        </a>
+        <div class="explore-online__text">
+          <h3>Virtual group tours</h3>
+          <p> Request a guided tour, reserve admission and lead your own small group, or request a live virtual tour led by a Museum guide.</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="contacts" id="contacts">
+      <h2>Contact us</h2>
+      <div class="contacts__container__wrapper">
+        <div class="contacts__container">
+          <div class="contacts__container--card">
+            <p class="contacts__title">The Met Fifth Avenue</p>
+            <div class="contacts__wrapper">
+              <p class="contacts__title--title">Phone</p>
+              <a href="tel:+1 212-535--7710" class="contacts__title--content">+1 212-535--7710</a>
+              <p class="contacts__title--title">Address</p>
+              <a href="https://maps.app.goo.gl/zW1hK1uyAj9jKZiR9" target="_blank">
+                <p>1000 Fifth Avenue <br>New York, NY 10028</p>
+              </a>
+            </div>
+          </div>
+          <div class="contacts__container--card">
+            <p class="contacts__title">The Met Cloisters</p>
+            <div class="contacts__wrapper">
+              <p class="contacts__title--title">Phone</p>
+              <a href="tel:+1 212-923-3700" class="contacts__title--content">+1 212-923-3700</a>
+              <p class="contacts__title--title">Address</p>
+              <a href="https://maps.app.goo.gl/Dd9D2kouF1f8HZ5r9" target="_blank">
+                <p>99 Margaret Corbin Drive<br>Fort Tryon Park<br>New York, NY 10040</p>
+              </a>
+            </div>
+          </div>
+        </div>
+        <form onsubmit="this.reset(); return false;">
```

Comment:
You could get rid of this line and change your button to type **reset**, would achieve a similar result

Reply:
I made like this because on the checklist says to do it. But if it's a similar result that's fine :D